ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/Formats.java

```diff
@@ -35,10 +36,13 @@
   public Formats(
       @JsonProperty(value = "keyFormat", required = true) final FormatInfo keyFormat,
       @JsonProperty(value = "valueFormat", required = true) final FormatInfo valueFormat,
-      @JsonProperty(value = "options", required = true) final Set<SerdeOption> options) {
+      @JsonProperty(value = "options") final Optional<Set<SerdeOption>> options
```

Comment:
Removed `required` as its acceptable for this set to be empty, (it generally is). 

When empty, it is no longer serialized to the JSON, hence it is now `Optional`.